### DISCUSSION TITLE
Github Actions Permissions

### DISCUSSION THREAD

**Discussion by OcelotWalrus** [Dungeons-of-Kathallion/Bane-Of-Wargs#18] [General]
Hey @lumbar527 , hope you're doing great!

So, I've made an github workflow for a continuous build for this game, but it requires at some point to create a new tag (release) and then push it to github. But this action requires github actions to have `read and write` access to this repository.

---
![image](https://github.com/Dungeons-of-Kathallion/Bane-Of-Wargs/assets/87318892/a3a33afc-881e-420d-948d-d42c88b57285)

---
![image](https://github.com/Dungeons-of-Kathallion/Bane-Of-Wargs/assets/87318892/d7faee74-e505-4f4f-918f-519a52200408)

---
So what I would like is that you change **in the organization settings** the permissions of the github actions. It would make me able to change it in the Bane Of Wargs repository.
Because even if the Bane Of Wargs repository has been created by me, I created it on the organization account, so settings like this can only change by the organization admins.

Thanks!

**Comment by lumbar527** [comment]
I think I changed it.

**Reply by OcelotWalrus** [reply]
Yes ! Thank you!
Have a great day by the way.

**Reply by lumbar527** [reply]
> Have a great day by the way.

You too!

**Comment by OcelotWalrus** [comment]
Hey @lumbar527 sorry to bother you again.

![image](https://github.com/Dungeons-of-Kathallion/Bane-Of-Wargs/assets/87318892/1527c18a-a5bf-4bf0-8ced-98ffcb68b37f)

I also need the PR thing enabled please ;)

Thanks

**Reply by lumbar527** [reply]
I checked the third option, was that what you wanted?

**Reply by OcelotWalrus** [reply]
Yes! Perfect! thanks.
